### ISSUE TITLE
Restore projects when opened

### ISSUE THREAD

**Issue by mpeterson2** [mpeterson2/save-session#115]
If you open a blank window, then open a project from there, files will not be restored. They should be.

**Comment by davidvartanian** [comment]
Hi, I write here just because the title of this issue. Is this package compatible with project-manager package? When I open a new file, write something and close Atom it works. When I open Atom again the unsaved file is there. But projects managed with project-manager package are not opened again (what Sublime does) and of course I lose all my unsaved changes on files related to that project. I've got Atom 0.209.0. Thanks!

**Comment by mpeterson2** [comment]
Yeah, that is the problem this issue is about. Once this is implemented, it should work with project-manager better.
